Monopoly.py

```diff
@@ -53,15 +53,15 @@
 sleep(2)
 # testing player to see if they understand
 while begin:
-    start = str(input("Are you ready to continue: "))
+    start = str(input("Are you ready to continue? "))
     if start.upper() != "YES":
         print("Oooops! Try again.")
         sleep(2)
     else:
         begin = False
-player_amount = int(input("How many players do you have: "))
+player_amount = int(input("How many players do you have? "))
```

Comment:
I think colon makes more sense.

Reply:
I don't because it's a question and questions have question marks

Reply:
But having a colon prompts the user to input something

Reply:
and a question prompts an answer

Reply:
it's really just preference Phillip should do whichever he prefers

Reply:
> it isn't very important. And it isn't very important anyway.